docs/LinuxWindowsSupport.md

```diff
@@ -15,7 +15,7 @@ Therefore, Linux and Windows support for the development environment is an ongoi
 
 As of **version 0.14** Android development with React native is mostly possible on Linux and Windows. You'll need to install [Node.js](https://nodejs.org/) 4.0 or newer. On Linux we recommend installing [watchman](https://facebook.github.io/watchman/docs/install.html), otherwise you might hit a node file watching bug.
 
-## What's missing on Windows
+## What's missing on Windows and Linux
 
 On Windows the packager won't be started automatically when you run `react-native run-android`. You can start it manually using:
```

Comment:
Should we get rid of "On Windows..." here and just the sentence with "The Packager..."?